crates/re_viewer/data/quick_start_guides/python_connect.md

```diff
@@ -12,26 +12,17 @@ Python package:
 pip install rerun-sdk
 ```
 
-### Try out the viewer
+### Logging your own data
 
-The Rerun SDK comes with a demo that can be used to try the viewer. You can send a demo recording
-to this viewer using the following command:
+Copy and paste the following snippet in a new Python file and execute it to create a new recording in this viewer:
 
-```sh
-python -m rerun_sdk --connect
+```python
+${EXAMPLE_CODE}
```

Comment:
Where does this come from?

Reply:
This is replaced by the appropriate code example, which is pulled from `crates/re_viewer/data/quick_start_{connect,spawn}.{cpp,py,rs}`, which are in turn symlink to the same files in `docs/code-examples` (required by `cargo publish`).